src/Sylius/Bundle/CartBundle/Controller/CartController.php

```diff
@@ -112,14 +111,29 @@ public function clearAction(Request $request)
     {
         $configuration = $this->requestConfigurationFactory->create($this->metadata, $request);
 
-        $eventDispatcher = $this->getEventDispatcher();
+        $this->isGrantedOr403($configuration, ResourceActions::DELETE);
+        $resource = $this->getCurrentCart();
```

Comment:
Is it the only difference between `ResourceController::deleteAction()` and this one?